src/GDBone2D.cpp

```diff
@@ -21,30 +21,25 @@ void GDBone2D::_bind_methods() {
 }
 
 Vector2 GDBone2D::get_bone_global_position(){
```

Comment:
This shows some precision issues due to the conversion between Dragonbones -> Godot coordinates.